ext/openssl/php_openssl.h

```diff
@@ -86,6 +86,11 @@ php_stream_transport_factory_func php_openssl_ssl_socket_factory;
 
 void php_openssl_store_errors();
 
+int php_openssl_cipher_iv_length(char *, zend_long *);
+int php_openssl_random_pseudo_bytes(zend_string **, zend_long);
+int php_openssl_encrypt(char *, size_t, char *, size_t, char *, size_t, zend_long, char *, size_t, zval *, zend_long, char *, size_t, zend_string **);
+int php_openssl_decrypt(char *, size_t, char *, size_t, char *, size_t, zend_long, char *, size_t, char *, zend_long, char *, size_t, zend_string **);
```

Comment:
These should probably be marked `PHPAPI` so they are usable from other DLLs on windows.